packages/schema/lib/schemas/FieldMetaSchema.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+
+module.exports = makeSchema({
+  id: '/FieldMetaSchema',
+  type: 'object',
+  description:
+    'Allows for additional metadata to be stored on the field. Only string, numeric or boolean values are allowed',
```

Comment:
Not necessary though

Reply:
The `additionalProperties` property for the schema means that this can be statically validated! For example, check out the error from running `zapier validate` with the following inputField:
```js
 {
        key: "my_key",
        label: "My Key",
        type: "string",
        required: false,
        meta: {
          subObject: { greeting: "hello!" },
        },
}
```

![image](https://github.com/user-attachments/assets/5bc9d98f-3455-4eb8-a7f0-79fc47cd1138)